mysql/repository-storage-running-low.yaml

```diff
@@ -0,0 +1,14 @@
+apiVersion: monitoring.appscode.com/v1alpha1
+kind: Runbook
+metadata:
+  namespace: monitoring
+  name: repository-storage-running-low
+spec:
+  diagnosticChecks:
+  remedyActions:
+    - type: IncreaseVolume
+      args:
+        description: Increase the volume size of repository backend
```

Comment:
We may need to add a target for this action. So target can be repository, db etc.